types/index.d.ts

```diff
@@ -96,9 +96,9 @@ declare namespace dayjs {
     locale(preset: string | ILocale, object?: Partial<ILocale>): Dayjs
   }
 
-  export type PluginFunc = (option: any, c: typeof Dayjs, d: typeof dayjs) => void
+  export type PluginFunc<T = unknown> = (option: T, c: typeof Dayjs, d: typeof dayjs) => void
```

Comment:
One last question, what's this <T = unknown> for?

Reply:
For compatibility and `dayjs.extend<T>()` as above, if a plugin ts definition doesn't pass a custom type for generic `T`, it fallbacks to `unknown`.